doc/rule-descriptions.md

```diff
@@ -24,7 +24,7 @@
 | [aria-required-children](https://dequeuniversity.com/rules/axe/4.2/aria-required-children?application=RuleDescription)           | Ensures elements with an ARIA role that require child roles contain them                                      | Critical          | cat.aria, wcag2a, wcag131                                                                  | failure, needs&nbsp;review | [ff89c9](https://act-rules.github.io/rules/ff89c9)                                                     |
 | [aria-required-parent](https://dequeuniversity.com/rules/axe/4.2/aria-required-parent?application=RuleDescription)               | Ensures elements with an ARIA role that require parent roles are contained by them                            | Critical          | cat.aria, wcag2a, wcag131                                                                  | failure                    | [bc4a75](https://act-rules.github.io/rules/bc4a75), [ff89c9](https://act-rules.github.io/rules/ff89c9) |
 | [aria-roledescription](https://dequeuniversity.com/rules/axe/4.2/aria-roledescription?application=RuleDescription)               | Ensure aria-roledescription is only used on elements with an implicit or explicit role                        | Serious           | cat.aria, wcag2a, wcag412                                                                  | failure, needs&nbsp;review |                                                                                                        |
-| [aria-roles](https://dequeuniversity.com/rules/axe/4.2/aria-roles?application=RuleDescription)                                   | Ensures all elements with a role attribute use a valid value                                                  | Serious, Critical | cat.aria, wcag2a, wcag412                                                                  | failure                    |                                                                                                        |
+| [aria-roles](https://dequeuniversity.com/rules/axe/4.2/aria-roles?application=RuleDescription)                                   | Ensures all elements with a role attribute use a valid value                                                  | Serious, Critical | cat.aria, wcag2a, wcag412                                                                  | failure, needs&nbsp;review |                                                                                                        |
```

Comment:
not sure if needed but this was an auto-generated change

Reply:
This happens sometimes when the file is out of date from develop. We can just include it as part of this commit it's fine.